app/experimenter/visualization/tests/api/constants.py

```diff
@@ -113,7 +113,7 @@ def get_test_data(cls):
         ABSOLUTE_METRIC_DATA_A = cls.get_absolute_metric_data(DATA_POINT_A)
         ABSOLUTE_METRIC_DATA_F = cls.get_absolute_metric_data(DATA_POINT_F)
         ABSOLUTE_METRIC_DATA_F_WITH_PERCENT = ABSOLUTE_METRIC_DATA_F.copy()
-        ABSOLUTE_METRIC_DATA_F_WITH_PERCENT.percent = 33.0
+        ABSOLUTE_METRIC_DATA_F_WITH_PERCENT.percent = 50.0
```

Comment:
How does this trigger the logic above?

Reply:
This segment is added: https://github.com/mozilla/experimenter/blob/e7892de3b55c4832df9bc2b19c46bf91487f0259/app/experimenter/visualization/tests/api/constants.py#L54 and it extends `DATA_IDENTITY_ROW` which is a user count metric: https://github.com/mozilla/experimenter/blob/e7892de3b55c4832df9bc2b19c46bf91487f0259/app/experimenter/visualization/tests/api/constants.py#L43

So we included it in the `total_population` count here: https://github.com/mozilla/experimenter/blob/e7892de3b55c4832df9bc2b19c46bf91487f0259/app/experimenter/visualization/api/v3/models.py#L73

And then when we did this division it caused the population % to be a 3rd because we included that segment: https://github.com/mozilla/experimenter/blob/e7892de3b55c4832df9bc2b19c46bf91487f0259/app/experimenter/visualization/api/v3/models.py#L83

When we exclude the segment, it goes to 50%.